_includes/templates/download.html

```diff
@@ -48,11 +48,19 @@ <h2>{{ page.latestversion }} {{CURRENT_RELEASE}} <a type="application/rss+xml" h
       <div>
         <img src="/assets/images/os/med_osx.png" alt="osx">
         <span>
-          <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.macdmg }}">macOS</a>
+          <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.macdmg }}">macOS (x86_64)</a>
           <span><a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.macdmg }}" class="dl" id="macdmg">dmg</a> -
             <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.mactar }}" class="dl" id="mactar">tar.gz</a></span>
         </span>
       </div>
+      <div>
+        <img src="/assets/images/os/med_osx.png" alt="osx" style="visibility:hidden">
```

Comment:
Why `visibility:hidden`?

Reply:
Intended to make invisible the image while having the image occupy the same block of space.

Although DOM-untidy, this tends to render OK-ish.

Reply:
Agree  it should be visible if it takes up another square

Reply:
force pushed a rev, same contents, just removing the visibility attr